changelog/changed-valid-access-ports-return-type.md

```diff
@@ -0,0 +1 @@
+`valid_access_ports` now returns a list of addresses rather than `GenericAp`
```

Comment:
This is private API IIUC so no changelog is needed.

Reply:
The impact of this is on the ArmCommunicationInterface and the sequences.